pages/mesosphere/dcos/services/edge-lb/1.5/tutorials/blue-green-deploy/index.md

```diff
@@ -1,192 +1,230 @@
 
-Below is a minimal Edge-LB configuration, `sample-minimal.json`, that load balances "svc-blue".
+    ```bash
+    dcos node list
+    ```
 
-```json
-{
-  "apiVersion": "V2",
-  "name": "sample-config",
-  "count": 1,
-  "haproxy": {
-    "frontends": [{
-      "bindPort": 80,
-      "protocol": "HTTP",
-      "linkBackend": {
-        "defaultBackend": "svc"
-      }
-    }],
-    "backends": [{
-      "name": "svc",
-      "protocol": "HTTP",
-      "services": [{
-        "marathon": {
-          "serviceID": "/svc-blue"
-          },
-          "endpoint": {
-            "portName": "web"
-          }
-      }]
-    }]
-  }
-}
-```
-
-Copy the example service and name it `svc-blue.json`.
-
-```json
-{
-    "id": "/svc-blue",
-    "cmd": "/start $PORT0",
-    "instances": 1,
-    "cpus": 0.1,
-    "mem": 32,
-    "container": {
-        "type": "DOCKER",
-        "docker": {
-            "image": "mesosphere/httpd"
-        }
-    },
-    "portDefinitions": [
-        {
-            "name": "web",
-            "protocol": "tcp",
-            "port": 0
-        }
-    ],
-    "healthChecks": [
-        {
-            "portIndex": 0,
-            "path": "/",
-            "protocol": "HTTP"
-        }
-    ]
-}
-```
-
-Make another copy of the example service and name it `svc-green.json`.
-
-```json
-{
-    "id": "/svc-green",
-    "cmd": "/start $PORT0",
-    "instances": 1,
-    "cpus": 0.1,
-    "mem": 32,
-    "container": {
-        "type": "DOCKER",
-        "docker": {
-            "image": "mesosphere/httpd"
-        }
-    },
-    "portDefinitions": [
-        {
-            "name": "web",
-            "protocol": "tcp",
-            "port": 0
-        }
-    ],
-    "healthChecks": [
-        {
-            "portIndex": 0,
-            "path": "/",
-            "protocol": "HTTP"
-        }
-    ]
-}
-```
+    For more information about how to find the IP address for a public agent, see [Finding a public agent IP](/mesosphere/dcos/2.0/administering-clusters/locate-public-agent/).
 
-1. Upload the configuration to Edge-LB.
+# Deploy the "green" version of the sample service
+1. Start deployment for the `svc-green.json` app definition:
 
-   ```
-   dcos edgelb create sample-minimal.json
+   ```bash
+   dcos marathon app add svc-green.json
    ```
 
-1. Start the service "svc-blue".
+1. Verify the blue and green versions of the sample service and the Edge-LB pool instance have been deployed successfully:
 
-   ```bash
-   dcos marathon app add svc-blue.json
-   ```
+    ```bash
+    dcos marathon app list
+    ```
+
+1. Modify the Edge-LB pool configuration file to point to `svc-green` by changing the `serviceID` setting.
 
-1. Now, "svc-blue" is exposed at `http://<public-ip>/`.
+         ```json
```

Comment:
Markdown formatting is broken here for `json` (because of the indentation I believe)

Reply:
Fixed.